test/unit/helpers.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:hmi_core/hmi_core.dart';
+import 'package:hmi_networking/hmi_networking.dart';
+
+List<DsDataPoint> decodeDataPoints(Uint8List bytes) {
+  return JdsLine.chunks(bytes, Jds.endOfTransmission)
+    .map((chunk) => String.fromCharCodes(chunk))
+    .where((rawPoint) => rawPoint.isNotEmpty)
+    .map((rawPoint) => const JsonCodec().decode(rawPoint) as Map<String, dynamic>)
+    .map((jsonEvent) => DsDataPoint(
+      type: DsDataType.fromString(jsonEvent['type'] as String),
+      path: jsonEvent['path'] as String,
+      name: jsonEvent['name'] as String,
+      value: jsonEvent['value'],
+      status: DsStatus.fromValue(jsonEvent['status']  as int),
+      history: jsonEvent['history'] as int? ?? 0,
+      alarm: jsonEvent['alarm'] as int? ?? 0,
+      timestamp: jsonEvent['timestamp'] as String,
+    ))
+    .toList();
+}
+
+Uint8List encodeDataPoints(List<DsDataPoint> dataPoints) {
+  return Uint8List.fromList(
+    dataPoints.map(
+      (dataPoint) => utf8.encode(dataPoint.toJson())
+        .toList()
+        ..add(Jds.endOfTransmission),
+    )
+    .expand((element) => element)
+    .toList(),
+  );
+}
+
+bool compareDataPoints(DsDataPoint first, DsDataPoint second) {
```

Comment:
let's implement it inside DsDataPoint

Reply:
issue added: #6

Reply:
Implemented in: https://github.com/a-givertzman/hmi_core/pull/11/files#diff-7a6ffd8685914ed6994bae425ab25cc79a02a60938f4c1f96861f0e9f5861de6

Old code removed: https://github.com/a-givertzman/hmi_networking/pull/3/commits/03d59003b40c1dfed3b08297a6cc116f8d33a01d#diff-7bfa8602f509100a884256af3c31a4205f042a8ea23df0900a1a06450f9c21ee